internal/node_loader.js

```diff
@@ -107,7 +107,6 @@ module.constructor._resolveFilename =
   var resolveLocations = [
     request,
     resolveRunfiles(request),
-    resolveRunfiles('TEMPLATED_workspace_name', 'TEMPLATED_label_package', request),
```

Comment:
I take it that removing this related to https://github.com/bazelbuild/rules_nodejs/issues/32?